sdk/core/core/inc/az_http_request_builder.h

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @file az_http_request_builder.h
```

Comment:
Please, remove unnecessary fields, such as @file, @author, @date. There were useful in the last century when there were no source control.